templates/email/bromley/_problem-confirm_extra.html

```diff
@@ -0,0 +1,4 @@
+By clicking validate you consent that for the purposes of investigating and
```

Comment:
Maybe we should prefix this with a `</p><p style="[% p_style %]">` so that is displays as a new paragraph in the HTML email? Looks a bit intimidating when it’s just appended to the existing paragraph!

<img width="308" alt="screen shot 2018-05-17 at 12 46 22" src="https://user-images.githubusercontent.com/739624/40175542-59dffdf0-59d0-11e8-8ec0-040562dc71cb.png">